cli/aws_orbit/cleanup.py

```diff
@@ -185,10 +185,11 @@ def delete_kubeflow_roles(env_stack_name: str, region: str) -> None:
             _logger.info(f"Removing role {role_name} - checking for attached policies")
             role_policies = iam_client.list_role_policies(RoleName=role_name).get("PolicyNames")
 
-            for policy in role_policies:
+            for policy_name in role_policies:
```

Comment:
why do we see these changes in your PR?  Something is not right yet.   I don't believe you have done these changes

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.detach_role_policy

After upgrading boto3 and botocore, have to change the parameter from name to arn.

Reply:
got it